### DISCUSSION TITLE
File Field | Prunable is always present when updating a file

### DISCUSSION THREAD

**Discussion by StanBarrows** [laravel/nova-issues#4740] [Q&A] [answered]
- Laravel Version: 9.*
- Nova Version: 4.13.*
- PHP Version: 8.1.*

### Description:

File Field | Prunable is always present when updating a file
- Tested without prunable method
- Tested with prunable(false)

![CleanShot 2022-08-24 at 20 03 34@2x](https://user-images.githubusercontent.com/10268813/186490781-09cf948c-15ea-4d9d-b122-ea34c01bcb9d.png)

```
   File::make(__('app/nova/fields.path'), 'path')
                ->disk(config('filesystems.default'))
                ->path(\App\Models\Template::PATH)
                ->hideFromIndex()
                ->prunable(false)
                ->rules('file', 'mimes:docx', 'max:2048')
                ->creationRules('required')
                ->updateRules('nullable'),
```

**Comment by crynobone** [comment, marked as answer]
You actually looking for `deletable(false)`. https://nova.laravel.com/docs/4.0/resources/file-fields.html#pruning-deletion

Pruning only runs when you delete a Model.
